queue/queue.go

```diff
@@ -123,6 +123,14 @@ var luaPop = redis.NewScript(2, `-- KEYS: [EpochMS QueueType]
 
 	-- if we didn't find one, try again from our bulk queue
 	if not result[1] or isFutureResult then
+		-- check if we are paused for bulk queue
+		local pausedBulkKey = "paused_bulk:" .. queueName
```

Comment:
maybe we call this `rate_limit_bulk` so we have a way to rate limit everything and a way to rate limit just bulk sends

Reply:
so replace `paused_bulk` with `rate_limit_bulk`?

Reply:
yeah